spa/src/common/Wrapper.tsx

```diff
@@ -25,19 +26,16 @@ const navigateToHomePage = () => {
 	});
 };
 
-export const Wrapper = (attr: {
-	children?: ReactNode | undefined;
-}) => {
-	return <WrapperOutterStyled>
+export const Wrapper = (attr: { children?: ReactNode | undefined }) => {
+	return (
+		<div css={wrapperOutterStyle}>
 			<Button
 				style={{ float: "right" }}
 				iconBefore={<CrossIcon label="Close" size="medium" />}
 				appearance="subtle"
-				onClick={ navigateToHomePage }
-			>
-		</Button>
-		<WrapperCenterStyled>
-			{ attr.children }
-		</WrapperCenterStyled>
-	</WrapperOutterStyled>;
+				onClick={navigateToHomePage}
+			></Button>
```

Comment:
/nit Can just close the component without the second closing tag `<Button ..... />` as there are no children

Reply:
Done